abb_irb120_support/urdf/irb120_macro.xacro

```diff
@@ -0,0 +1,195 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+<xacro:macro name="abb_irb120" params="prefix">
+	<!-- link list -->
```

Comment:
Please change the indentation to two spaces per level. I know the other xacros don't do this (yet), but we should try to do this in the new ones. I've used [expand](http://linux.die.net/man/1/expand) for this earlier.